stl/inc/yvals_core.h

```diff
@@ -1565,7 +1566,11 @@ _EMIT_STL_ERROR(STL1004, "C++98 unexpected() is incompatible with C++23 unexpect
 
 #if _HAS_CXX20
 #if !defined(__EDG__) || defined(__INTELLISENSE__) // TRANSITION, GH-395
-#define __cpp_lib_concepts 202002L
+#if _HAS_CXX23 // TRANSITION, GH-395 - move down to "macros with language mode sensitivity" section
+#define __cpp_lib_concepts 202207L // P2404R3 Move-Only Types For Comparison Concepts
+#else // ^^^ C++23 / C++20 vvv
+#define __cpp_lib_concepts 202002L // P2408R5 Ranges Iterators As Inputs To Non-Ranges Algorithms

```

Comment:
According to [SD-FeatureTest `__cpp_lib_concepts`](https://isocpp.org/std/standing-documents/sd-6-sg10-feature-test-recommendations#__cpp_lib_concepts), the previous value `202002L` indicated `P1964R2 Replacing boolean With boolean-testable`.

(The only value for [`__cpp_lib_algorithm_iterator_requirements`](https://isocpp.org/std/standing-documents/sd-6-sg10-feature-test-recommendations#__cpp_lib_algorithm_iterator_requirements), `202207L`, indicates WG21-P2408R5.)